README.md

```diff
@@ -55,6 +55,11 @@ generic with minimal need for project-specific behavior.
 
 - AWS credentials configured in `~/.aws/credentials` and/or `~/.aws/config`
 
+- [git-secrets] to help prevent secrets (passwords, credentials, etc.) from
+  being submitted into the git repository. It is recommended you follow the
```

Comment:
```suggestion
  being submitted into the git repository. It is recommended you follow the
```

Recommended means optional. What level of protection does one get if one skips this step? The requirement is that `git secrets` is enabled for all clones of our repositories and that it prevents the user from making commits containing text that matches the general patterns as well as the specific AWS-related patterns.  Specifically, I am doubtful that the default steps enable the AWS-specific patterns. You should be specific on how a person satisfies that requirement. Ideally, this section should also specify a way to verify the correct functioning of `git secrets`. If the bullet item gets too long, it would be OK to extract it out into a separate section and link to it here.